.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["es2015", "react"],
-  "plugins": ["transform-object-rest-spread"]
+  "presets": ["cf"],
```

Comment:
Noooooooo we don't want to maintain cf preset anymore before an internal style guide for ES6 emerge